lib/hydra/pcdm.rb

```diff
@@ -14,6 +14,7 @@ module PCDM
     autoload :Collection,             'hydra/pcdm/models/collection'
     autoload :Object,                 'hydra/pcdm/models/object'
     autoload :File,                   'hydra/pcdm/models/file'
+    autoload :VersionedFile,          'hydra/pcdm/models/versioned_file'
```

Comment:
PCDM::VersionedFile is just a variation of PCDM::File and really it's a stopgap for handling a deficiency in ActiveFedora.  If PCDM::File changes, PCDM::VersionedFile must change too, so I put them together.  

Really I think we shouldn't need a separate class at all to indicate that a File is versionable — it should rely on the :versionable flag on the fedora resource itself, provide methods for setting/reading that flag on all instances of File, and extend the instance behaviors accordingly when the File has been marked as versionable.  

If we were to update AF::File in that way, we would need to update PCDM::File to have that behavior and delete VersionedFile.  I want that change to happen in one repo so it's clear what's happening.